<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>einstellung</class>
 <widget class="QDialog" name="einstellung">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
font: &quot;Courier New&quot;;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>15</y>
     <width>396</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>36</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;Courier New&quot;;
color: rgb(240, 240, 240);</string>
   </property>
   <property name="text">
    <string>System</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>331</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Courier New&quot;;
color: rgb(240, 240, 240);</string>
      </property>
      <property name="text">
       <string>Lüfter:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Courier New</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Courier New&quot;;
color: rgb(240, 240, 240);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Raum&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_fan_netz">
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Courier New&quot;;
color: rgb(240, 240, 240);</string>
      </property>
      <property name="text">
       <string>45%</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_9">
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Courier New&quot;;
color: rgb(240, 240, 240);</string>
      </property>
      <property name="text">
       <string>Temp.:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="label_fan_raum">
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Courier New&quot;;
color: rgb(240, 240, 240);</string>
      </property>
      <property name="text">
       <string>20%</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Courier New</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Courier New&quot;;
color: rgb(240, 240, 240);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Platine&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_fan_plat">
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Courier New&quot;;
color: rgb(240, 240, 240);</string>
      </property>
      <property name="text">
       <string>66%</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_temp_plat">
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Courier New&quot;;
color:rgb(255,255,0);</string>
      </property>
      <property name="text">
       <string>35°C</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Courier New</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Courier New&quot;;
color: rgb(240, 240, 240);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Netzteile&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label_temp_raum">
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Courier New&quot;;
color:rgb(255,127,0);</string>
      </property>
      <property name="text">
       <string>27°C</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_temp_netz">
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Courier New&quot;;
color:rgb(0,255,0);</string>
      </property>
      <property name="text">
       <string>30°C</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_13">
      <property name="font">
       <font>
        <family>Courier New</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Courier New&quot;;
color: rgb(240, 240, 240);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Pi CPU&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="label_temp_pi">
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Courier New&quot;;
color:rgb(0,255,0);</string>
      </property>
      <property name="text">
       <string>43.2°C</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLabel" name="label_15">
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Courier New&quot;;
color: rgb(240, 240, 240);</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>Courier New</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Courier New&quot;;
color: rgb(240, 240, 240);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;PC&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QLabel" name="label_fan_pc">
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Courier New&quot;;
color: rgb(240, 240, 240);</string>
      </property>
      <property name="text">
       <string>0%</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="label_temp_pc">
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Courier New&quot;;
color:rgb(0,255,0);</string>
      </property>
      <property name="text">
       <string>43.2°C</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <family>Courier New</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Courier New&quot;;
color: rgb(240, 240, 240);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Onkyo&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLabel" name="label_temp_onkyo">
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Courier New&quot;;
color:rgb(0,255,0);</string>
      </property>
      <property name="text">
       <string>43.2°C</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QLabel" name="label_fan_onkyo">
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Courier New&quot;;
color: rgb(240, 240, 240);</string>
      </property>
      <property name="text">
       <string>100%</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>400</y>
     <width>71</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="bilder.qrc">
     <normaloff>:/bilder/Icons/arrow_left_icon.png</normaloff>:/bilder/Icons/arrow_left_icon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Off">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="bilder.qrc">
     <normaloff>:/bilder/Icons/shutdown-icon.png</normaloff>:/bilder/Icons/shutdown-icon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>70</width>
     <height>70</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="bilder.qrc"/>
 </resources>
 <connections/>
</ui>
